README.md

```diff
@@ -2,42 +2,98 @@ PReview Code frontend [![Build Status](https://travis-ci.org/preview-code/fronte
+ 
+		curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+		sudo apt-get install nodejs -y
+2. install the _latest_ version of [npm](https://www.npmjs.com/):
 
-# Installation for serviceWorker
-Service-worker requires you to run the website on `https`. Therefore you need
-to install a local self-signed certificate for `localhost:5200`.
+		sudo npm install npm@latest -g
+3. install [git](https://git-scm.com/):
 
-When running `gulp serve` by default the certificate in `certificate/` is
-loaded and attached. However, when opening the website you will get a message
-that the page is insecure. Perform the following steps to resolve this:
+		sudo apt-get install git -y
+__Please note__: It is important to first install `nodejs`, as shown here, and then `npm`. The script for `nodejs` downloads an older version of `npm`, which we subsequently update.
 
-(This guide is inspired by http://stackoverflow.com/a/15076602/2761676)
+## Install tooling
+* Install Chrome:
 
-1. Open the website, you should see an insecurity warning from Chrome
-* Click on advanced
-* Click on continue anyways
-* Click on the red lock in the url bar
-* Click `Details`
-* Click `View certificate`
-* Click `Details`
-* Click `Export...`
+  	Please visit https://www.google.com/chrome/ to install the newest version of Chrome.
+
+## Install the npm (global) packages
+1. install the [java](http://packages.ubuntu.com/en/trusty/default-jre) runtime
+
+		sudo apt-get install default-jre
+1. install [bower](https://bower.io/)
+
+		sudo npm install -g bower
+2. install [gulp](http://gulpjs.com/)
+		
+        sudo npm install -g gulp-cli
+3. install [Polymer CLI](https://www.polymer-project.org/1.0/)
+		
+        sudo npm install -g polymer-cli
+4. install [polyserve](https://github.com/PolymerLabs/polyserve)
+		
+        sudo npm install -g polyserve
+5. install [web-component-tester](https://github.com/Polymer/web-component-tester)
+		
+        sudo npm install -g web-component-tester@4.2.2
+5. install [web-component-tester-istanbul](https://github.com/thedeeno/web-component-tester-istanbul)
+		
+        sudo npm i -g web-component-tester-istanbul@git://github.com/TimvdLippe/web-component-tester-istanbul.git#coverage-log-test-result-fix
+__Please note__: version `4.3.5` of `web-component-tester` causes errors. Please use `4.2.2` until further notice.
+
+## Clone the frontend
+1. Clone the repository
+		
+        git clone https://github.com/preview-code/frontend.git
+2. Enter the (newly created) directory named `frontend`
+
+## Prepare the frontend to run
+1. Install the (local) npm packages
+		
+        sudo npm install -only=dev
+2. Install bower packages
+		
+        bower install
+__Please note__: It is important to first install the `npm` packages and then
```

Comment:
This is only true for global packages, since you installed these already it is not a requirement.